model_helen_comments.py

```diff
@@ -0,0 +1,127 @@
+"""Models and database functions for showme db."""
+
+from flask_sqlalchemy import SQLAlchemy
+
+db = SQLAlchemy()
+
+
+##############################################################################
+
+class Show(db.Model):
+    """Runway show details."""
+
+    __tablename__ = "shows"
+
+    show_id = db.Column(db.Integer,
+                        autoincrement=True,
+                        primary_key=True)
+    season = db.Column(db.String(6))
+    year = db.Column(db.Integer)
+    brand_id = db.Column(db.Integer,
+                         db.ForeignKey('brands.brand_id'))
+
+    # Define relationship to brands
+    brands = db.relationship('Brand')
+
+    def __repr__(self):
+        return "<Show show_id=%s season=%s brand_id=%s year=%s>" % (
+            self.show_id,
+            self.season,
+            self.brand_id,
+            self.year)
+
+
+class Show_Color(db.Model):
+    """Association table connecting specific color info to shows."""
```

Comment:
In case they didn't cover this in class, the relationship between `Show` and `Color` is called "many-to-many", because a show can have many colors and a color can have many shows.  If you want to, here's how to use your association table so that you can directly call `show_object.colors` or `color_object.shows`, and avoid querying the show_color table. http://docs.sqlalchemy.org/en/latest/orm/basic_relationships.html#many-to-many
Maybe this should just be food for thought, because your code seems like its working well without it, and sometimes complex sqlalchemy relationships can end up having weird side effects.